app.js

```diff
@@ -1,3 +1,65 @@
-/**
- * Created by Dima on 10.09.2016.
- */
+const http = require('http');
+const coldFilmParsers = require('./coldFilmParsers');
+const utf8 = require('utf8');
+const co = require('co');
+
+function getRequest(path,parser) {
+    return new Promise((resolve, reject) => {
+        let options = {
+            hostname: 'coldfilm.ru',
+            port: 80,
+            method: 'GET',
+            path
+        };
+        let request = http.request(options, (res) => {
+            let body = [];
+            res
+                .on('data', (chunk) => {
+                    body.push(chunk);
+                })
+                .on('end', () => {
+                    body = Buffer.concat(body).toString();
+                    let data = parser(body);
+                    resolve(data);
+                })
+        });
+
+        request.on('error', (err) => {
+            reject(err);
+        });
+
+        request.end();
+
+    });
+}
+let name = 'Мистер робот';
+
+function* getSerial() {
+    let path = '/search/?q=' + utf8.encode(name).split(' ').join('+') + ';t=0;p=1;md=news';
+    let series = [];
+    while (true) {
```

Comment:
i think that you can make loop condition better :)
don't forget that there is not only `while() {}`
`do {} wile()` exist too